e2e/package.json

```diff
@@ -6,11 +6,11 @@
   "private": true,
   "license": "BSD",
   "scripts": {
-    "test": "cross-env jest --maxWorkers=3",
+    "test": "cross-env jest --maxWorkers=5",
     "test:debugTest": "cross-env PUPPETEER_HEADLESS=false DEBUG=true node --inspect-brk node_modules/.bin/jest",
     "test:debug": "cross-env PUPPETEER_HEADLESS=false DEBUG=true jest --detectOpenHandles",
-    "test-local": "cross-env WORKBENCH_ENV=local DEBUG=true jest --maxWorkers=2",
-    "test-local:debug": "cross-env WORKBENCH_ENV=local PUPPETEER_HEADLESS=false DEBUG=true jest --maxWorkers=2",
+    "test-local": "cross-env WORKBENCH_ENV=local DEBUG=true jest --maxWorkers=3",
+    "test-local:debug": "cross-env WORKBENCH_ENV=local PUPPETEER_HEADLESS=false DEBUG=true jest --maxWorkers=3",
```

Comment:
Server should be able to handle more concurrent tests. Test playback time is faster.